cmd/really/CMakeLists.txt

```diff
@@ -11,4 +12,4 @@ set_target_properties(really
     PROPERTIES
         CXX_STANDARD 20
         CXX_STANDARD_REQUIRED YES
-        CXX_EXTENSIONS ON)
\ No newline at end of file
+        CXX_EXTENSIONS ON)
```

Comment:
For portability, we should not turn this on.  While portability is not critical now, it might be later.  What extensions are we using we cannot live without?

Reply:
Will turn it off for now.

Reply:
I'll leave this as is for now since it's everywhere.  Let's clean this up if needed in all the CMakeLists.txt files.